docker/docker-compose-8.11.1.yml

```diff
@@ -0,0 +1,157 @@
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      - "ELASTICSEARCH_URL=http://elasticsearch:9200"
+    ports:
+      - "5601:5601"
+    depends_on:
+      - "elasticsearch"
+    networks:
+      - esnet
+
+  # Logstash
+  logstash:
+    build: ./logstash/8.11.1
+    environment:
+      - "ELASTICSEARCH_URL=elasticsearch:9200"
+    ports:
+      - 8100:8100
+    volumes:
+      - ./logstash/8.11.1/pipeline/logstash-userale.conf:/usr/share/logstash/pipeline/logstash-userale.conf
+      - ./logstash/8.11.1/templates/userale.json:/usr/share/logstash/templates/userale.json
+    depends_on:
+      - "elasticsearch"
+    networks:
+      - esnet
+
+  # Distill
+  distill:
+     build: ./distill
+     ports:
+       - 8090:8090
+     depends_on:
+       - "elasticsearch"
+     networks:
+       - esnet
+
+  # TAP's Database
+  db:
+     build: ./db
+     image: senssoft/postgres:latest
+     ports:
+       - "5432:5432"
+
+  # TAP
+  tap:
+     build: ./tap
+     command: bash -c "/usr/src/app/wait-for-postgres.sh"
+     ports:
+       - "8010:8010"
+     depends_on:
+       - db
+     links:
+       - db:db
+
+  # Specific container and configuration to monitor the HOST (filesystem, memory, processes,...)
+  metricbeat:
+    image: docker.elastic.co/beats/metricbeat:8.11.1
+#    restart: unless-stopped
+    user: root
+    pid: host
+    volumes:
+      #The commented sections below enable Metricbeat to monitor the Docker host
+      #rather than the Metricbeat container. These are used by the system module.
+      - /proc:/hostfs/proc:ro
+      - /sys/fs/cgroup:/hostfs/sys/fs/cgroup:ro
+      #We mount the host filesystem so we can report on disk usage with the system module
+      - /:/hostfs:ro
+      #Mount the metricbeat configuration so users can make edits
+      - ./beats/metricbeat/metricbeat.yml:/usr/share/metricbeat/metricbeat.yml
+      #Mount the modules.d directory into the container. This allows user to 
+      #potentially make changes to the modules and they will be dynamically loaded.
+      - ./beats/metricbeat/modules.d/:/usr/share/metricbeat/modules.d/
+      #Allows us to report on docker from the hosts information
+      - /var/run/docker.sock:/var/run/docker.sock
+    depends_on:
+      - elasticsearch
+    environment:
+      - ELASTICSEARCH_URL=http://elasticsearch:9200
+      - KIBANA_URL=http://kibana:5601
+      - LOGSTASH_URL=http://logstash:9600
+      - POSTGRESQL_URL="postgresql://db:5432/tapdb?sslmode=disable"
+      - POSTGRESQL_USERNAME=tapuser
+      - POSTGRESQL_PASSWORD="Dr@p3rUs3r"
+    extra_hosts:
+      - "elasticsearch:172.17.0.1" # The IP of docker0 interface to access host from container
+      - "kibana:172.17.0.1" # The IP of docker0 interface to access host from container
+      - "logstash:172.17.0.1"
+      - "db:172.17.0.1"
+    command: metricbeat -e -system.hostfs=/hostfs -strict.perms=false
+    network_mode: host # Mandatory to monitor HOST filesystem, memory, processes,...
+
+networks:
```

Comment:
This is giving a warning, and has a simple fix.

`network esnet: network.external.name is deprecated. Please set network.name with external: true`